pom.xml

```diff
@@ -121,8 +123,8 @@
                     <artifactId>maven-compiler-plugin</artifactId>
                     <version>3.5.1</version>
                     <configuration>
-                        <source>1.8</source>
-                        <target>1.8</target>
+                        <source>${maven.compiler.source}</source>
+                        <target>${maven.compiler.target}</target>

```

Comment:
@MahmoudQassass can you paste a screen shot of a `mvn clean install` that's clean on your machine?

I tried on my machine, and it fails!

Reply:
Yes it fails beacuse missing plugin, I will add it to pom.xml.